src/tests/labels.test.ts

```diff
@@ -0,0 +1,19 @@
+import { newCustomer } from '../test_utils'
+const customer = newCustomer()
+
+describe('Labels', () => {
+    it('can create a new label', async () => {
+        const rn = await customer.labels.create(
+            {
+                name: 'label-name',
+                text_label: {
+                    background_color: '#e993eb',
+                    description: 'description',
+                },
+            },
+            { validate_only: true }
+        )
+        // Empty as validate only is true
+        expect(rn.results).toEqual([])
```

Comment:
would it be better to use `rn.request.partialFailure` to test success?

Reply:
`partialFailure` is only valid for multiple mutations, so in the case of a single mutation (like this) validate only is a good test.

Reply:
If an error occurs the whole test block will fail as well 👌